perl.h

```diff
@@ -6003,14 +6003,23 @@ typedef void (*XSINIT_t) (pTHX);
 typedef void (*ATEXIT_t) (pTHX_ void*);
 typedef void (*XSUBADDR_t) (pTHX_ CV *);
 
-/* TODO: find somewhere to store this */
 enum Perl_custom_infix_precedence {
-    INFIX_PREC_LOW  =  10, /* non-associative */
-    INFIX_PREC_REL  =  30, /* non-associative, just below `==` */
-    INFIX_PREC_ADD  =  50, /* left-associative, same precedence as `+` */
-    INFIX_PREC_MUL  =  70, /* left-associative, same precedence as `*` */
-    INFIX_PREC_POW  =  90, /* right-associative, same precedence as `**` */
-    INFIX_PREC_HIGH = 110, /* non-associative */
+    /* These numbers are spaced out to give room to insert new values as
```

Comment:
If the values here don't have much or any negative impact on memory or speed of the compiled program it might be nice to space them out even more. That way, maybe the actual values themselves would never need to be changed because there'd be enough room inbetween for plugins to do things?

Reply:
Yeah there's plenty of space, and they only have to fit in an `enum`, which is probably `int`-sized. I'll space them out more.

Reply:
Done. Though I've kept them within the range of a U8, in case in future we want to split the field up and add some flags in the remaining bits.